docs/developer/HootenannyBuild.asciidoc

```diff
@@ -3,77 +3,88 @@
 
 Hootenanny uses a top level Makefile to control the building and testing of code from all three codebases: Hootenanny Core (C++), Hootenanny Web Services (Java), and Hootenanny User Interface (Javascript).  Underneath, Hootenanny also uses qmake and Maven for additional build capabilities.
 
-Type 'make help' to see a list of all possible build and configuration options, some of which are described in this section.
+Type +make help+ to see a list of all possible build and configuration options, some of which are described in this section.
 
 Before building Hootenanny, the application must be configured:
+
 ----------------
 aclocal && autoconf && autoheader && automake && ./configure <options>
 ----------------
 
 Once Hootenanny is initially configured, this command only need be re-run if the underlying make files change in source control:
+
 ------------------
 aclocal && autoconf && autoheader && automake
 ------------------
 
 This line needs to be re-run everytime you change configuration options:
+
 -----------------
 ./configure <options>
-------------------
+-----------------
 
 At the time of this writing, the Hootenanny configuration options consist of:
-* --with-services - builds all Hootenanny Core services database code as well as all Hootenanny Java Web Services code.
-* --with-uitests - enables the Hootenanny User Interface tests
-* --with-rnd - builds all Hootenanny Core experimental R&D code
-* --with-coverage - generates code test coverage reports
+
+* +--with-services+ - builds all Hootenanny Core services database code as well as all Hootenanny Java Web Services code.
+* +--with-uitests+ - enables the Hootenanny User Interface tests
+* +--with-rnd+ - builds all Hootenanny Core experimental R&D code
+* +--with-coverage+ - generates code test coverage reports
```

Comment:
Most of these changes are formatting to make things look better in the final document and in `vim`.  Some things confused the highlighting mechanism so I fixed them.